drivers/hmis/app/models/hmis/ce/match/candidate.rb

```diff
@@ -0,0 +1,41 @@
+# an eligible, prioritized client for a given candidate pool
+module Hmis::Ce::Match
+  class Candidate < GrdaWarehouseBase
+    self.table_name = 'ce_match_candidates'
+    belongs_to :candidate_pool, class_name: 'Hmis::Ce::Match::CandidatePool'
+    belongs_to :client, class_name: 'Hmis::Hud::Client'
+
+    # FIXME: permissions
+    scope :viewable_by, ->(_user) { all }
+
+    # order by descending priority, NULL values last. Use id as a tie-breaker
+    scope :prioritized, -> {
+      order(
+        Arel::Nodes::Descending.new(arel_table[:priority_score]).nulls_last,
+        arel_table[:id].asc,
+      )
```

Comment:
I'm probably missing something, but can't this just be:
```suggestion
      order(
        arel_table[:priority_score].desc.nulls_last,
        arel_table[:id].asc,
      )
```

Reply:
That's better, thanks!